docs/develop/auth.md

```diff
@@ -77,6 +76,27 @@ func AuthHandler(ctx context.Context, token string) (auth.UID, error) {
 }
 ```
 
+
+## Returning custom error messages on auth handlers
```

Comment:
This feels too specific; I think what we want to communicate is more along the lines of "here's how to signal that the request isn't authenticated".

How about:

```suggestion
## Handling auth errors
```